lib/cog/chat/hipchat/template_processor.ex

```diff
@@ -24,6 +24,8 @@ defmodule Cog.Chat.HipChat.TemplateProcessor do
     do: "<code>#{text}</code>"
   defp process_directive(%{"name" => "fixed_width_block", "text" => text}),
     do: "<pre>#{text}</pre>"
+  defp process_directive(%{"name" => "link", "text" => text, "url" => url}),
```

Comment:
derp, I was wrong @christophermaier. That second head shouldn't ever get called. When links are specified using the markdown syntax they always have both text and url fields.